modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -82,6 +82,68 @@ body .ddm-user-view-content {
 	padding: 0;
 }
 
+.lfr-ddm-help-text-flex-container{
+	display: flex;
+	justify-content: space-between;
+	.lfr-ddm-help-text-flex-element{
+		margin: auto 0;
+		flex-shrink: 1;
+	}
+}
+
+.lfr-ddm-help-text-content{
+	margin-top: 8px;
+	margin-bottom: 8px;
+	.lfr-ddm-help-text-line{
+		margin-top: 8px;
+		margin-bottom: 8px;
+	}
+}
+
+.lfr-ddm-help-text-label{
+	color: #272833;
+	cursor: default;
+	display: inline-block;
+	font-size: 0.875rem;
+	font-weight: 600;
+	max-width: 100%;
+	word-wrap: break-word;
+	margin-bottom: 0px;
+}
+
+.help-text-button{
```

Comment:
@mateusgomes01 can you please order the names of the classes alphabetically?

Reply:
I'll do now. I think I forgot to run `gradlew format`